doc-files/release-notes-24.md

```diff
@@ -0,0 +1,221 @@
+[JDK-8341440](https://bugs.openjdk.org/browse/JDK-8341440) | ScrollPane: no immediate effect changing fitWidth/fitHeight | controls
+[JDK-8341687](https://bugs.openjdk.org/browse/JDK-8341687) | Memory leak in TableView after interacting with TableMenuButton | controls
+[JDK-8342233](https://bugs.openjdk.org/browse/JDK-8342233) | Regression: TextInputControl selection is backwards in RTL mode | controls
+[JDK-8344067](https://bugs.openjdk.org/browse/JDK-8344067) | TableCell indices may not match the TableRow index | controls
+[JDK-8347305](https://bugs.openjdk.org/browse/JDK-8347305) | RichTextArea Follow-up | controls
+[JDK-8347715](https://bugs.openjdk.org/browse/JDK-8347715) | RichTextArea Follow-up: Minor Bugs | controls
+[JDK-8348736](https://bugs.openjdk.org/browse/JDK-8348736) | RichTextArea clamp and getText | controls
+[JDK-8323787](https://bugs.openjdk.org/browse/JDK-8323787) | Mac System MenuBar throws IOB exception | graphics
+[JDK-8333374](https://bugs.openjdk.org/browse/JDK-8333374) | Cannot invoke "com.sun.prism.RTTexture.contentsUseful()" because "this.txt" is null | graphics
+[JDK-8336097](https://bugs.openjdk.org/browse/JDK-8336097) | UserAgent Styles using lookups are promoted to Author level if look-up is defined in Author stylesheet | graphics
+[JDK-8336389](https://bugs.openjdk.org/browse/JDK-8336389) | Infinite loop occurs while resolving lookups | graphics
+[JDK-8339068](https://bugs.openjdk.org/browse/JDK-8339068) | [Linux] NPE: Cannot read field "firstFont" because "<local4>" is null | graphics
+[JDK-8340405](https://bugs.openjdk.org/browse/JDK-8340405) | JavaFX shutdown hook can hang preventing app from exiting | graphics
+[JDK-8341010](https://bugs.openjdk.org/browse/JDK-8341010) | TriangleMesh.vertexFormat Property default value is wrong | graphics
+[JDK-8341418](https://bugs.openjdk.org/browse/JDK-8341418) | Prism/es2 DrawableInfo is never freed (leak) | graphics
+[JDK-8342703](https://bugs.openjdk.org/browse/JDK-8342703) | CSS transition is not started when initial value was not specified | graphics
+[JDK-8183521](https://bugs.openjdk.org/browse/JDK-8183521) | Unable to type characters with tilde with swiss german keyboard layout | localization
+[JDK-8336277](https://bugs.openjdk.org/browse/JDK-8336277) | Colors are incorrect when playing H.265/HEVC on Windows 11 | media
+[JDK-8336938](https://bugs.openjdk.org/browse/JDK-8336938) | Update libFFI to 3.4.6 | media
+[JDK-8336939](https://bugs.openjdk.org/browse/JDK-8336939) | Update Glib to 2.80.4 | media
+[JDK-8336940](https://bugs.openjdk.org/browse/JDK-8336940) | Update GStreamer to 1.24.6 | media
+[JDK-8338701](https://bugs.openjdk.org/browse/JDK-8338701) | Provide media support for libavcodec version 61 | media
+[JDK-8346228](https://bugs.openjdk.org/browse/JDK-8346228) | Update GStreamer to 1.24.10 | media
+[JDK-8346229](https://bugs.openjdk.org/browse/JDK-8346229) | Update Glib to 2.82.4 | media
```

Comment:
I'm not sure it's beneficial to include obsolete version updates. If the final update of a dependency is to 1.2.3, then any previous update (1.2.1) doesn't need to be listed, in my opinion.

Reply:
I am in favor of keeping intermediary revisions for the same reason @kevinrushforth mentioned, and also because it eliminates manual filtering (?) and associated mistakes.

Reply:
I would say they're not needed.  It could be confusing I suppose (you may at first glance think GStreamer is now version  On the other hand, I can understand wanting to list all the JDK issues that were involved in a release.

Reply:
> I am in favor of keeping intermediary revisions for the same reason @kevinrushforth mentioned, and also because it eliminates manual filtering (?) and associated mistakes.

Actually, I was pointing out that it might be more consistent to _not_ keep them -- treating them like transiently introduced bugs -- but I can see both points. I'll let this percolate for a day or two and then we can decide.

By default I'll leave them in since that's what we've done in the past, but they add little value.

Reply:
I decided to remove these two bugs. In addition to avoiding confusion, listing those two releases, which have publishes CVEs logged against them, will leave a mistaken impression that JavaFX is vulnerable to the bugs associated with those two older versions.

To address the filtering issues I defined two new labels: `javafx-rn-exclude` and `javafx-rn-include`, which can be applied to bugs and are used to filter in the same way that the `noreg-*` labels already are. The filter now has no manual list of bugs to include or exclude.